runtime/cudaq/platform/default/rest/helpers/braket/BraketExecutor.h

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
```

Comment:
nit: add emacs mode

Reply:
Addressed in PR https://github.com/NVIDIA/cuda-quantum/pull/2448